### DISCUSSION TITLE
How can i fix a Argument `id` is missing error.

### DISCUSSION THREAD

**Discussion by umutkamanoglu** [prisma/prisma#25744] [Q&A] [answered]
### Question

hello i get "Argument `id` is missing." error with next.js and postgresql. how can i fix it.

my schema:
```prisma
model users {
  id              Int    @id @default(autoincrement())
  username        String
  password        String
  email           String
  profile_picture String
  role            Int    @default(0)
}
```
my code:
```js
const user = await prisma.users.create({
        data: {
          username,
          email,
          password: hashedPassword,
        },
      });
```
### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hello @umutkamanoglu 

From your schema, i can see you have `id` field with `@id @default(autoincrement())`, which should automatically generate an ID for new records. Did you regenerate your Prisma Client to ensure it's up to date with your schema?
```
npx prisma generate
```

**Reply by umutkamanoglu** [reply]
this solved my problem. thank you very much.
